### ISSUE TITLE
[Bug] NoSuchFieldError: XML_TAG_ICON in LayoutBlockCompletionContributor.java:34

### ISSUE THREAD

**Issue by Bashev** [magento/magento2-phpstorm-plugin#1218]
**Describe the bug** (*)

```
java.lang.NoSuchFieldError: XML_TAG_ICON
	at com.magento.idea.magento2plugin.completion.provider.LayoutBlockCompletionContributor.addCompletions(LayoutBlockCompletionContributor.java:34)
	at com.intellij.codeInsight.completion.CompletionProvider.addCompletionVariants(CompletionProvider.java:25)
	at com.intellij.codeInsight.completion.CompletionContributor.fillCompletionVariants(CompletionContributor.java:157)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributor(CompletionService.java:83)
	at com.intellij.codeInsight.completion.impl.CompletionServiceImpl.getVariantsFromContributor(CompletionServiceImpl.java:315)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributors(CompletionService.java:75)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:154)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:146)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:142)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:132)
	at com.intellij.codeInsight.completion.XmlNoVariantsDelegator.fillCompletionVariants(XmlNoVariantsDelegator.java:12)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributor(CompletionService.java:83)
	at com.intellij.codeInsight.completion.impl.CompletionServiceImpl.getVariantsFromContributor(CompletionServiceImpl.java:315)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributors(CompletionService.java:75)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:154)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:146)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:142)
	at com.intellij.codeInsight.template.impl.LiveTemplateCompletionContributor$1.addCompletions(LiveTemplateCompletionContributor.java:91)
	at com.intellij.codeInsight.completion.CompletionProvider.addCompletionVariants(CompletionProvider.java:25)
	at com.intellij.codeInsight.completion.CompletionContributor.fillCompletionVariants(CompletionContributor.java:157)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributor(CompletionService.java:83)
	at com.intellij.codeInsight.completion.impl.CompletionServiceImpl.getVariantsFromContributor(CompletionServiceImpl.java:315)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributors(CompletionService.java:75)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:154)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:146)
	at com.intellij.codeInsight.completion.CompletionResultSet.runRemainingContributors(CompletionResultSet.java:142)
	at com.intellij.codeInsight.completion.XmlCompletionContributor$3.addCompletions(XmlCompletionContributor.java:130)
	at com.intellij.codeInsight.completion.CompletionProvider.addCompletionVariants(CompletionProvider.java:25)
	at com.intellij.codeInsight.completion.CompletionContributor.fillCompletionVariants(CompletionContributor.java:157)
	at com.intellij.codeInsight.completion.XmlCompletionContributor.fillCompletionVariants(XmlCompletionContributor.java:185)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributor(CompletionService.java:83)
	at com.intellij.codeInsight.completion.impl.CompletionServiceImpl.getVariantsFromContributor(CompletionServiceImpl.java:315)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributors(CompletionService.java:75)
	at com.intellij.codeInsight.completion.CompletionService.getVariantsFromContributors(CompletionService.java:58)
	at com.intellij.codeInsight.completion.CompletionService.performCompletion(CompletionService.java:134)
	at com.intellij.codeInsight.completion.BaseCompletionService.performCompletion(BaseCompletionService.java:43)
	at com.intellij.codeInsight.completion.impl.CompletionServiceImpl.performCompletion(CompletionServiceImpl.java:334)
	at com.intellij.codeInsight.completion.CompletionProgressIndicator.lambda$calculateItems$11(CompletionProgressIndicator.java:870)
	at com.intellij.util.indexing.FileBasedIndex.lambda$ignoreDumbMode$0(FileBasedIndex.java:202)
	at com.intellij.openapi.util.RecursionManager$1.computePreventingRecursion(RecursionManager.java:114)
	at com.intellij.util.indexing.FileBasedIndexEx.ignoreDumbMode(FileBasedIndexEx.java:723)
	at com.intellij.util.indexing.FileBasedIndex.ignoreDumbMode(FileBasedIndex.java:201)
	at com.intellij.util.indexing.DumbModeAccessType.ignoreDumbMode(DumbModeAccessType.java:43)
	at com.intellij.codeInsight.completion.CompletionProgressIndicator.calculateItems(CompletionProgressIndicator.java:866)
	at com.intellij.codeInsight.completion.CompletionProgressIndicator.runContributors(CompletionProgressIndicator.java:854)
	at com.intellij.codeInsight.completion.CodeCompletionHandlerBase.lambda$startContributorThread$6(CodeCompletionHandlerBase.java:373)
	at com.intellij.codeInsight.completion.AsyncCompletion.lambda$tryReadOrCancel$5(CompletionThreading.java:172)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1086)
	at com.intellij.codeInsight.completion.AsyncCompletion.tryReadOrCancel(CompletionThreading.java:170)
	at com.intellij.codeInsight.completion.CodeCompletionHandlerBase.lambda$startContributorThread$7(CodeCompletionHandlerBase.java:365)
	at com.intellij.codeInsight.completion.AsyncCompletion.lambda$startThread$0(CompletionThreading.java:95)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:190)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:592)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:667)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:623)
	at com.intellij.ope
```

**To Reproduce** (*)

**Expected behavior** (*)

**Screenshots**

**Please complete the following information:** (*)

 - OS: Windows 11 10.0
 - PhpStorm/Intellij version: 2022.3 EAP
 - Plugin Version: 4.4.0

**Additional context**

**Comment by makzef** [comment]
@magento I am working on this

**Comment by makzef** [comment]
Hello @Bashev.
Thank you for this report.

This is one of the compatibility issue with PhpStrom 2022.3 EAP. 
Magento plugin was available for this version of IDE by mistake. That has already been corrected.
The plugin is not currently supported for EAP version.
This will change when PhpStorm 2022.3 becomes stable.

Thank you.
CC @bohdan-harniuk

**Comment by bohdan-harniuk** [comment]
Hello, @Bashev!

We will try to deliver fix for the 2022.3 EAP in the 4.4.1.
Issue will be fixed in the https://github.com/magento/magento2-phpstorm-plugin/issues/1224

Regards,
